components/Buttons/src/ColorThemer/MDCButtonColorThemer.h

```diff
@@ -31,23 +31,26 @@
 + (void)applySemanticColorScheme:(nonnull id<MDCColorScheming>)colorScheme
                         toButton:(nonnull MDCButton *)button;
 
+
 /**
- Applies a color scheme to theme to an MDCFlatButton.
+ Applies a color scheme to theme to an text button.
 
  @param colorScheme The color scheme to apply to @c flatButton.
- @param flatButton An MDCFlatButton instance to apply a color scheme.
+ @param textButton An MDCButton instance to apply a color scheme.
  */
 + (void)applySemanticColorScheme:(nonnull id<MDCColorScheming>)colorScheme
-                    toFlatButton:(nonnull MDCButton *)flatButton;
+                    toTextButton:(nonnull MDCButton *)textButton;
 
 /**
- Applies a color scheme to theme to an MDCRaisedButton.
+ Applies a color scheme to theme to an contained button.
 
- @param colorScheme The color scheme to apply to @c raisedButton.
- @param raisedButton An MDCRaisedButton instance to apply a color scheme.
+ @param colorScheme The color scheme to apply to @c flatButton.
+ @param containedButton An MDCFlatButton instance to apply a color scheme.
  */
 + (void)applySemanticColorScheme:(nonnull id<MDCColorScheming>)colorScheme
-                  toRaisedButton:(nonnull MDCButton *)raisedButton;
+               toContainedButton:(nonnull MDCButton *)containedButton;
```

Comment:
Should we provide individual classes for these APIs as well, mimicking the work done in the ButtonThemer?

Reply:
I was going to delay that since there are "will deprecate" apis being used by this class

Reply:
If you think it's right to move in that direction then let's just move there without introducing an intermediary API.

Reply:
oh ok. i will do that then. That will have to happen tomorrow.